internal/attestation/attestation.go

```diff
@@ -35,6 +35,7 @@ var (
 type Attestation interface {
 	Type() string
 	PredicateType() string
+	ContentType() string
```

Comment:
Is this still needed? Seems to not be used.

Reply:
We don't output it, so it can be removed, it goes unstated that attestations can be other than JSON so...